### PR TITLE
Added test to /factorial endpoint.

### PR DESCRIPTION
**Pull request by iaansanity**

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -23,5 +23,23 @@ describe("Starting tests for app.js ->", () => {
...
+      .post("/factorial")
+      .type("form")
+      .send({"number": 9})
+      .end((err, res) => {
+        if (err) {
+          done(err);
+          process.exit(1);
+        } else {
+          res.should.have.status(200);
+          res.body.should.have.property('result');
```
Comment (ZiyanK) [inline tests/tests.js:38]: Add validation for the logic as well. For example factorial of 9 is 362880. So check if the response consists of this output. 
Hint: use res.body.should.equal

**Review by ZiyanK — CHANGES_REQUESTED** [review]
Kindly make the necessary change. Everything else lgtm.

File: tests/tests.js (hunk 1 above)
Comment (iaansanity) [reply tests/tests.js:64]: Fixed conflict and added script for checking the output. Kindly check. Thanks!

**Review by ZiyanK — APPROVED** [review]
Looks good to me

**Pull request merged**